exercises/chapter6/test/Solutions.purs

```diff
@@ -0,0 +1,49 @@
+module Test.Solutions where
```

Comment:
Specifying the file for solutions will enable easier comparison of solutions, similar to [what's available](https://exercism.io/tracks/purescript/exercises/pangram/solutions) in exercism.io